source/state_representation/include/state_representation/trajectory/CartesianTrajectory.hpp

```diff
@@ -0,0 +1,145 @@
+#pragma once
+
+#include "state_representation/space/cartesian/CartesianPose.hpp"
+#include "state_representation/space/cartesian/CartesianState.hpp"
+#include "state_representation/trajectory/TrajectoryBase.hpp"
+
+namespace state_representation {
+
+/**
+ * @class CartesianTrajectoryPoint
+ * @brief Struct to represent a Cartesian trajectory point
+ */
+struct CartesianTrajectoryPoint : public TrajectoryPoint {};
```

Comment:
Is this necessary or just nice? I'm just wondering. And while we are here, you might as well define a constructor from a duration and CartesianState, that will help reduce code in the source file

Reply:
Yes, I think this is just nicer to have a dedicates struct for this class which could also allow us to more easily specify fields if ever needed.

Also, my original understanding (because of the attribute name) was that JointStates have a robot name, not necessarily a state name, that is common to all. In that case, we would only need to store a name for Cartesian states. But still, I find it a bit nicer to keep it there for future use